### PR TITLE
Chore/deploy updates

### PR DESCRIPTION
**Pull request by davidpmacneil**

### CODE REVIEW

File: provisioning/database/database.bicep [hunk 1]
```diff
@@ -33,9 +33,6 @@ resource privateDNSzoneForDatabase 'Microsoft.Network/privateDnsZones@2020-06-01
 }
 
 resource databaseServer 'Microsoft.DBforMySQL/flexibleServers@2021-05-01' = {
-  dependsOn: [
-    privateDNSzoneForDatabase
-  ]
```
Comment (evanjamesjackson) [inline provisioning/database/database.bicep:38]: Did you successfully deploy without this? I know VS Code complains but I was experiencing issues actually getting it to deploy properly without it

Comment (davidpmacneil) [reply provisioning/database/database.bicep:38]: It wouldn't let me get past the database deploy portion with that at all

Comment (evanjamesjackson) [reply provisioning/database/database.bicep:38]: Oh really it was throwing an error?

Comment (davidpmacneil) [reply provisioning/database/database.bicep:38]: Yea, it's too far up my terminal now, but you can see a warning in the file in vscode if you have it open

Comment (evanjamesjackson) [reply provisioning/database/database.bicep:38]: Interesting - it it works without it though then perfect

Comment (davidpmacneil) [reply provisioning/database/database.bicep:38]: yea can verify after containers are up

**Review by evanjamesjackson — APPROVED** [review]
Nice - thank you for fixing

**Pull request merged**